src/workflow/unlock_bip39.c

```diff
@@ -92,14 +92,7 @@ workflow_t* workflow_unlock_bip39(void (*callback)(void* param), void* callback_
     return result;
 }
 
-static void _unlock_cb(void* param)
-{
-    (void)param;
-    workflow_blocking_unblock();
-}
-
 void workflow_unlock_bip39_blocking(void)
 {
-    workflow_stack_start_workflow(workflow_unlock_bip39(_unlock_cb, NULL));
```

Comment:
I'm now looking at where `workflow_unlock_bip39` is used and it seems only unlock.c is calling it but since it's removed in https://github.com/digitalbitbox/bitbox02-firmware/pull/542, how about deleting `workflow_unlock_bip39`?

Reply:
Yeah it's coming, got a big queue of deletion commits :P

Reply:
:+1: